node-operators/networks/collator.md

```diff
@@ -351,45 +344,43 @@ curl http://127.0.0.1:9933 -H \
 
 - **mapping** (*address* optionalAuthorID) —— 将显示所有储存在链上的映射内容，或是根据您的输入显示相关内容
 
-### 映射外部信息 {: #mapping-extrinsic }
+### 映射Extrinsic {: #mapping-extrinsic } 
 
-如果您想要将您的author ID映射至您的账户，您需要成为[候选人池](#become-a-candidate)中的一员。当您成功成为候选人，您将需要传送您的映射外部信息（交易）。请注意，每一次注册author ID将会绑定Token。为此，请执行以下步骤：
+如果您想要将您的author ID映射至您的账户，您需要成为[候选人池](#become-a-candidate)中的一员。当您成功成为候选人，您将需要传送您的映射extrinsic（交易）。请注意，每一次注册author ID将会绑定Token。为此，请执行以下步骤：
 
  1. 进入**Developer**标签
+  2. 点击**Extrinsics**
 
- 2. 点击**Extrinsics**
-
- 3. 选取您想要映射author ID的目标账户（用于签署此交易）
+  3. 选取您想要映射author ID的目标账户（用于签署交易）
 
- 4. 选择**authorMapping**函数
+  4. 选择**authorMapping**
 
- 5. 选择**addAssociation()**函数
+  5. 选择**addAssociation()**函数
 
- 6. 输入author ID。在本示例中，可在前一个部分通过RPC调用`author_rotateKeys`获得
+  6. 输入author ID。在本示例中，可在前一个部分通过RPC调用`author_rotateKeys`获得
 
- 7. 点击**Submit Transaction**
+   7. 点击**Submit Transaction**
 
 ![Author ID Mapping to Account Extrinsic](/images/node-operators/networks/collators/collator-polkadotjs-4.png)
 
-如果交易成功，您将看到确认通知。如果没有，请确认您是否已加入[候选人池](#become-a-candidate)。
+如果交易成功，您将在屏幕上看到确认通知。如果没有，请确认您是否已加入[候选人池](#become-a-candidate)。
 
 ![Author ID Mapping to Account Extrinsic Successful](/images/node-operators/networks/collators/collator-polkadotjs-5.png)
 
-### 检查映射设定 {: #checking-the-mappings }
+### 检查映射设定 {: #checking-the-mappings } 
 
 您也可以通过验证链上状态来确认目前的链上映射情况。为此，请执行以下步骤：
 
  1. 进入**Developer**标签
+  2. 点击**Chain state**
 
- 2. 点击**Chain state**
-
- 3. 选择**authorMapping**作为查询状态
+  3. 选择**authorMapping**作为查询状态
 
- 4. 选择**mappingWithDeposit**函数
+  4. 选择**mappingWithDeposit**函数
 
- 5. 提供author ID进行查询。您也可以通过关闭按钮以停止检索所有链上的映射情况
+  5. 提供author ID进行查询。您也可以通过关闭按钮以停止检索所有链上的映射情况
 
- 6. 点击**+**按钮来传送RPC调用
+   6. 点击**+**按钮来传送RPC调用
 
 ![Author ID Mapping Chain State](/images/node-operators/networks/collators/collator-polkadotjs-6.png)
```

Comment:
After this there should be a section on Join the Discord: https://github.com/PureStake/moonbeam-docs/pull/229/files#diff-4a1e759dec4fe268e80b04913929273d86c98ee78cb33411197d028e1b3fb5b0R345-R355